bin/xdmod-akrr-ingestor

```diff
@@ -327,7 +327,7 @@ function cleanup_and_exit($exitCode = 0)
         unset($appKernelIngestor);
     }
 
-    if ($logger instanceof \Log) {
+    if ($logger instanceof \Psr\Log\LoggerInterface) {
```

Comment:
Any idea why this if statement checks for instanceof ? Seems a bit odd to me. Under what circumstances would it not be a log class that is different from the default?

Reply:
Not sure to be honest, I'll ask @nsimakov

Reply:
I would say, to skip closing logger if it creation failed